debezium-connector-mongodb/src/test/java/io/debezium/connector/mongodb/transforms/ExtractNewDocumentStateTest.java

```diff
@@ -237,4 +247,39 @@ public void shouldFailWhenTheSchemaLooksValidButDoesNotHaveTheCorrectFields() {
 
         assertThat(transformed).isNull();
     }
+
+    @Test
+    @FixFor("DBZ-1430")
+    @SkipWhenKafkaVersion(check = LESS_THAN, value = KAFKA_241, description = "Kafka throws NullPointerException prior to 2.4.1")
```

Comment:
LOL, feeling kinda guilty now about having done this change upstream, seeing the cost here ;)